runner/runners/invoke.go

```diff
@@ -129,6 +134,7 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 		downloadTimer = inv.stat.Latency(stats.WorkerDownloadLatency_ms).Time()
 		inv.stat.Counter(stats.WorkerDownloads).Inc(1)
 	}
+	rts.checkoutStart = time.Now()
```

Comment:
Just a thought - setting these timing fields sprinkled throughout the codebase could end up being somewhat difficult to find. 

How would you feel about making the runTimes fields (e.g. invokeStart, invokeEnd, etc...) some custom struct with a time.Time & creating some method on that custom struct type that sets the value to time.Now()? Would allow for easier grep-ability.

Reply:
Yeah... I added a crude alias around time.Now that at least makes these grep-able (I guess, the only way to strictly enforce would be to break out in a separate pkg with non-exported field names). Added a note that I'd like to end up breaking the invoker down from the current "one big do-it function" style, that would allow for these all to get recorded once in a sane place.